services/src/libs/sentry.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * The sentry module is responsible for passing messages along to
+ * the sentry server.
+ */
+
+const logger    = require('winston');
+const Raven     = require('raven');
+
+
+class Sentry {
+  static initialize(sentryUrl) {
+    if (!sentryUrl) {
+      logger.error('No sentry url defined.');
+      return;
+    }
+    Raven.config(sentryUrl).install();
+  }
+
+  static sendOutput(data) {
```

Comment:
What is the expected behavior if `sendOutput` is invoked without `initiatialize` having run successfully?

Reply:
if initialize is not called, then the entire module throws errors.  It's a part of the application startup.